lib/clickhouse-activerecord/schema_dumper.rb

```diff
@@ -56,6 +56,7 @@ def table(table, stream)
 
           # detect view table
           match = sql.match(/^CREATE\s+(MATERIALIZED\s+)?VIEW/)
+          match_live = sql.match(/^CREATE LIVE VIEW/)
```

Comment:
Why not using a generic variable `is_view = sql.match(/^CREATE\s+((MATERIALIZED|LIVE)\s+)?VIEW/)`?

Reply:
because in the next line there where 

> tbl.print ', materialized: true' if match && match[1].presence

At the moment I decided to add another variable instead of improving the regex + the condition for `tbl.print ', materialized: true'`